dbms/tests/queries/0_stateless/00910_client_window_size_detection.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+. $CURDIR/../shell_config.sh
+
+function cleanup() { rm -f -- -; }
+trap cleanup EXIT
+
+set -e
+set -o pipefail
+
+${CLICKHOUSE_CLIENT} --help </dev/null | wc -L
+script -e -q -c "${CLICKHOUSE_CLIENT} --help" - </dev/null >/dev/null
```

Comment:
What is the real purpose of this statement?

Reply:
BTW it is Ok if the test will fail badly (infinite loop, segfault) in failure case - you don't have to filter...

Reply:
you mean the `>/dev/null` part, I think that saving output of the program that prints `\n` endlessly (for 600 seconds, according to default timeout) is very noisy (plus it will be saved to a file), but I'm ok to drop that part

Reply:
And also in this case `.reference` should contain the whole `--help` output, which will break the test once new option will be added